src/main/resources/Help/Code/Translations.xml

```diff
@@ -0,0 +1,251 @@
+help.macros.tableOfContents.title=Table of Contents
+help.macros.thisIsABoxMessage=This is a box message
+help.macros.thisIsAnInformativeMessage=This is an informative message
+help.macros.thisIsASuccessMessage=This is a success message
+help.macros.timeSeriesChart=Time Series Chart
+help.macros.title=Macros
+help.macros.titleOfTheBox=Title of the box
+help.macros.thereAreSeveralSpecialisedBoxMacros=There are several specialised box macros: 
+help.macros.thisIsAnErrorMessage=This is an error message
+help.macros.thisIsAWarningMessage=This is a warning message
+help.macros.tree.compactPageHierarchyTree=Compact Page Hierarchy Tree
+help.macros.tree.description={0}Page Tree Macro{0} displays the hierarchy of pages.
+help.macros.tree.fullEntityTree=Full Entity Tree
+help.macros.tree.pageHierarchyTree=Page Hierarchy Tree
+help.macros.tree.title=Tree
+help.macros.tree.wikiSpace=Wiki {0} Space {0} Page Tree
+help.macros.useMacro.description=Macros are reusable pieces of functionality that can be integrated inside pages.
+help.macros.useMacro.tip=The {0} WYSIWYG Editor{1} allows you to insert macros and to customize their parameters when editing pages. Look for the {2} button on the tool bar.
+help.macros.useMacro.title=Use Macros
+help.macros.viewMoreMacros=View more macros
+help.macros.warningMacro=Warning Macro
+help.mainXwikiFeatures.description=Congratulations for acquiring your XWiki instance. XWiki is very versatile and can be used to create knowledge bases, intranets or any type of business application you want. This is achievable because of XWiki's customization power, that starts at page level, from a simple to a more advanced usage.
+help.mainXwikiFeatures.title=Main XWiki features
+help.pageEditing.description=Use a powerful {0}WYSIWYG{1} content editor or {2}syntax{3} to create rich wiki pages.
+help.pageEditing.title=Page Editing
+help.seeExamples.label=See examples
+help.templates.article.description=Article page
+help.templates.article.myNewArticle=My new article
+help.templates.article.title=Article
+help.templates.contents=Contents
+help.templates.references=References
+help.templates.paragraph=Paragraph
+help.templates.simple=Simple
+help.templates.subParagraph=Sub-paragraph
+help.templates.subSubParagraph=Sub-sub paragraph
+help.templates.createYourTemplate.description=Integrate structure and macros in an existing page, and transform it in a template.
+help.templates.createYourTemplate.tip={0}Template Providers{1}  expose templates on the Create Page dialog so that the users can use them.
+help.templates.createYourTemplate.title=Create your template
+help.templates.description=Templates are predefined pages that are copied when creating a new page.
+help.templates.encyclopedia.credits=Credits
+help.templates.encyclopedia.description=Encyclopedia page
+help.templates.encyclopedia.falabella=Falabella
+help.templates.encyclopedia.image=Image
+help.templates.encyclopedia.miniatureHorse=Miniature horse
```

Comment:
same comment about "miniatureHorse". The translations keys could have more generic titles.